src/components/WorkOrder/WorkOrderDetails.tsx

```diff
@@ -66,6 +73,24 @@ const WorkOrderDetails = (props: Props) => {
 
   const currentWorkOrderActionMenu = workOrderActionMenu()
 
+  const fetchAlerts = async () => {
+    const alertsResponse = await getAlerts(property.propertyReference)
+
+    if (!alertsResponse.success) {
+      setAlertsError(alertsResponse.error.message)
+      setAlertsLoading(false)
+      return
+    }
+
+    setAlerts(alertsResponse.response.alerts)
+    setAlertsLoading(false)
+  }
+
+  useEffect(() => {
+    setAlertsLoading(true)
+    fetchAlerts()
+  }, [])
```

Comment:
would it be cleaner to move `setAlertsLoading(true)` to the top of the method? It would be more self contained